### PR TITLE
Add NotificationHubs schema

### PR DESCRIPTION
**Pull request by daschult**

### CODE REVIEW

File: schemas/2016-03-01/Microsoft.NotificationHubs.json [hunk 1]
```diff
@@ -0,0 +1,686 @@
...
+          ]
+        },
+        "apiVersion": {
+          "type": "string",
+          "enum": [
+            "2016-03-01"
+          ]
+        },
+        "properties": {
+          "oneOf": [
```
Comment (daschult) [inline schemas/2016-03-01/Microsoft.NotificationHubs.json]: Yes, I do. I'll have to look and see why my changes didn't get your SKU changes.

**Comment by daschult** [comment]
@TianoMS @wenwu449 Is anything stopping us from merging this pull request?

**Comment by TianoMS** [comment]
@daschult seems you have a comment about sku. Is there a pending change on it?

**Comment by TianoMS** [comment]
@daschult Any update on this one? Seems there's a comment you are working on.

**Pull request closed without merging**